spec/policies/order_policy_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe OrderPolicy do
+  subject { described_class.new(user, order) }
+
+  let(:order) { Order.create }
```

Comment:
See if we can use a factory here instead of `Order.create`. So it would look like this:
```ruby
let(:order) { create(:order) }
```
If that makes it crash, just go ahead and revert to the current solution.